generators/tests/org/jetbrains/kotlin/generators/tests/GenerateTests.kt

```diff
@@ -1105,6 +1105,7 @@ fun main(args: Array<String>) {
                 val pattern = "^([\\w\\-_]+)\\.kt$"
                 model("quickfix/modifiers", pattern = pattern, filenameStartsLowerCase = true, recursive = false)
                 model("quickfix/override/typeMismatchOnOverride", pattern = pattern, filenameStartsLowerCase = true, recursive = false)
+                model("quickfix/variables/changeMutability", pattern = pattern, filenameStartsLowerCase = true, recursive = false)
```

Comment:
Just curious why it's not `quickfix/override/changeMutability`. I guess it would include, e.g., var-to-val quick fix, maybe? Are there any counterpart you refer?

Reply:
Oh, is this a directory name under `testData`?